pages/platform/ai/training_tuto_02_build_custom_image/guide.en-gb.md

```diff
@@ -122,51 +314,70 @@ docker build . -t <image-identifier>
 > `docker buildx build --platform linux/amd64 ...`
 >
 
-## Test it locally (Optional)
+**Try to find a name that is easily identifiable**. This will allow you to manage your Docker images more easily, especially when you have multiple images and versions.
+
+For example, we could use: 
+
+``` {.console}
+docker build . -t <cnn_image_segmentation_project>
+```
 
-If you want to verify that your built image is working properly, run the following command :
+## Test the image locally (Optional)
+
+If we want to verify that our built image is working properly, we can run the following command:
 
 ``` {.console}
 docker run --rm -it --user=42420:42420 <image-identifier>
 ```
 
 > [!warning]
 >
-> Don't forget the `--user=42420:42420` argument if you want to simulate the exact same behavior that will occur on **AI TRAINING jobs**. It executes the docker container as the specific OVHcloud user (user **42420:42420**).
+> Don't forget the `--user=42420:42420` argument if you want to simulate the exact same behavior that will occur on **AI Training jobs**. It executes the docker container as the specific OVHcloud user (user **42420:42420**).
 
-## Push image in the registry of your choice
+## Push the image in the registry of our choice
 
-Pushing your image to a registry is needed in order for AI Training to pull it.
+Pushing our image to a registry is needed in order for AI Training to pull it.
 
-AI Training provides a default registry called **Shared registry** where users are able to push their custom images. It is linked with every project by default.
+AI Training provides, for each Public Cloud project, a default registry called **shared registry** where all users of a same Public Cloud project are able to push their custom images.
 
-If you prefer using your own private docker registry instead of the shared one, feel free to use it. Just don't forget to [add your registry in your AI Training project](https://docs.ovh.com/gb/en/publiccloud/ai/training/add-private-registry) before using it.
+> [!warning]
+>
+> This shared registry can help you perform your tests, but **should not be used in production**, as we reserve the right to delete its content if deemed necessary.
+> This is why it can be interesting to add other registries. Learn how to do it by following [this documentation](LINK).
```

Comment:
Hello @MathieuBsqt , it seems a link is missing here.

Reply:
Oops, well done. Just corrected it, thanks !